README.md

```diff
@@ -99,20 +94,25 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#MetisTuner">Metis Tuner</a></li>
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#BOHB">BOHB</a></li>
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#GPTuner">GP Tuner</a></li>
-          </ul>
-          <li><b>Tuner for <a href="docs/en_US/AdvancedFeature/GeneralNasInterfaces.md">NAS</a></b></li>
-          <ul>
-          <li><a href="docs/en_US/Tuner/BuiltinTuner.md#NetworkMorphism">Network Morphism</a></li>
-          <li><a href="examples/tuners/enas_nni/README.md">ENAS</a></li>
-          </ul>
         </ul>
           <a href="docs/en_US/Assessor/BuiltinAssessor.md">Assessor</a>
           <ul>
-          <ul>
           <li><a href="docs/en_US/Assessor/BuiltinAssessor.md#Medianstop">Median Stop</a></li>
           <li><a href="docs/en_US/Assessor/BuiltinAssessor.md#Curvefitting">Curve Fitting</a></li>   
           </ul>
-          </ul>  
+          <a href="docs/en_US/Compressor/Overview.md">Model Compression (Alpha)</a>
+          <ul>
+          <li><a href="docs/en_US/Compressor/Pruner.md#agp-pruner">AGP Pruner</a></li>
+          <li><a href="docs/en_US/Compressor/Quantizer.md#qat-quantizer">QAT Quantizer</a></li>
+          <li><a href="docs/en_US/Compressor/Quantizer.md#dorefa-quantizer">DoReFa Quantizer</a></li>
+          </ul>
+          <a href="https://github.com/microsoft/nni/blob/dev-nas-refactor/docs/en_US/NAS/Overview.md">NAS (Alpha)</a>
+          <ul>
+          <li><a href="docs/en_US/Tuner/BuiltinTuner.md#NetworkMorphism">Network Morphism</a></li>
+          <li><a href="examples/tuners/enas_nni/README.md">ENAS</a></li>
+          <li><a href="https://github.com/microsoft/nni/tree/dev-nas-refactor/examples/nas/darts">Darts</a></li>
+          <li><a href="docs/en_US/Tuner/BuiltinTuner.md#PPOTuner">PPO</a></li>
```

Comment:
Why PPO is a NAS tuner but Random tuner is not? I think they all based on the same method to prase search space?